app/containers/Message.jsx

```diff
@@ -6,16 +6,15 @@ import styles from 'css/components/message';
 
 const cx = classNames.bind(styles);
 
-const Message = ({message, type, dismissMessage}) => (
-    <div className={cx('message', {
-      show: message && message.length > 0,
-      success: type === 'SUCCESS'
-    })} onClick={dismissMessage}>{message}</div>
+const Message = ({style, text, dismissMessage}) => (
+    <div className={cx('message', [style, {show: text && text.length > 0}])}
+      onClick={dismissMessage}
+    >{text}</div>
 );
 
 Message.propTypes = {
-  message: PropTypes.string,
-  type: PropTypes.string,
+  style: PropTypes.string,
```

Comment:
ah, excellent point - `type` kinda indicates data type or something. I have seen success/warning/error passed in as boolean props, though that can get regretably busy if there's lots of options. It does look tidy though...

```
<Message text="Good news!" success />
```